src/stories/Library/Forms/input/DoubleInputRow.tsx

```diff
@@ -0,0 +1,30 @@
+import Input from "./Input";
+
+export type DoubleInputRowProps = {
+  leftLabel: string;
+  rightLabel: string;
+};
+
+const DoubleInputRow: React.FC<DoubleInputRowProps> = ({
+  leftLabel,
+  rightLabel,
+}) => {
+  return (
+    <div className="contact-info-flex">
+      <Input
+        id={leftLabel}
+        label={leftLabel}
+        type="text"
+        classNames="patron__input patron__input--desktop mr-16"
+      />
+      <Input
+        id={rightLabel}
+        label={rightLabel}
+        type="text"
+        classNames="patron__input patron__input--desktop"
```

Comment:
Same comment as the previous "patron__input" is specifc compare to the palcement/naming of the `DoubleInputRow` component

Reply:
And the answer is also same as above.